modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector-design-center.adoc

```diff
@@ -0,0 +1,67 @@
+= Microsoft Dynamics 365 - Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+The Flow Designer feature of Design Center enables you to create apps visually. To use Flow Designer, 
+work with your Anypoint Platform administrator to ensure that you have 
+a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. 
+For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Flow Designer field values, see 
+the xref:microsoft-dynamics-365/microsoft-dynamics-365-connector-reference.adoc[Microsoft Dynamics 365 Connector Reference]. 
+
+// Ensure that the connector reference xref and title are correct.
+
+== Configure the Input Source Trigger
+
+. In Design Center, click *Create*.
+. Click *Create new application* to open Flow Designer.
+. Specify a value for *Project name*, and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*. 
+. Click the trigger card.
+. If you are using your connector as an input source trigger, search for the connector; 
+otherwise, you can use HTTP or Scheduler.
```

Comment:
Do you complete these fields for all of the input trigger? If not, you should say which trigger requires these fields.